test/test_helper.rb

```diff
@@ -231,10 +257,22 @@ class TemplateRenderer
   def initialize(osfamily, template)
     @osfamily = osfamily
     @template = template
+    @ksvalidator_cmd = ENV['KSVALIDATOR'] || '/usr/bin/ksvalidator'
```

Comment:
We install ksvalidator via pip in our CI so it's not in `/usr/bin` but is in `$PATH`.

Reply:
Damn, good catch.